sys/arm64/arm64/trap.c

```diff
@@ -110,6 +111,7 @@ static abort_handler *abort_handlers[] = {
 	[ISS_DATA_DFSC_CAP_SEALED] = cap_abort,
 	[ISS_DATA_DFSC_CAP_BOUND] = cap_abort,
 	[ISS_DATA_DFSC_CAP_PERM] = cap_abort,
+	[ISS_DATA_DFSC_LC_SC] = cap_mmu_abort,
```

Comment:
Should probably only be installed for Morello kernels... but same goes for the others

Reply:
Yeah.